src/main/java/com/iuc/requests/controller/UserController.java

```diff
@@ -19,15 +18,26 @@
 
     @Autowired
     private UserService userService;
+    @Autowired
+    ModelMapper modelMapper;
 
     @GetMapping("/staff")
     public StaffDto findStaffByEmail(@RequestParam(value = "email") @Email String email) {
-        return null;
+        if(email.isEmpty()) {
```

Comment:
Not need to do it. Because @Email already do the work for you.

Reply:
L'observation a ete pris en compte.